### PR TITLE
Make each column scrollable, break on words in snippets

### PR DESCRIPTION
**Pull request by zephraph**
Fixes [PURCHASE-1908](https://artsyproduct.atlassian.net/browse/PURCHASE-1908)

Ensures that each panel in a conversation scrolls individually. 

**Before**
fix-layout-before

**After**
fix-layout-after

### CODE REVIEW

File: src/v2/Apps/Conversation/Routes/Conversation/index.tsx [hunk 1]
```diff
@@ -16,12 +16,24 @@ import {
...
 import { DetailsFragmentContainer as Details } from "../../Components/Details"
+import styled from "styled-components"
 interface ConversationRouteProps {
   me: Conversation_me
   conversationID: string
   match: Match
 }
 
+const ConstrainedHeightFlex = styled(Flex)`
+  height: calc(100vh - 145px);
```
Comment (xtina-starr) [inline src/v2/Apps/Conversation/Routes/Conversation/index.tsx:27]: Just curious what does the `145px` represent?

Comment (xtina-starr) [reply src/v2/Apps/Conversation/Routes/Conversation/index.tsx:27]: Oh is that accounting for the header thing that contains Inbox, Conversation with..., etc?

File: src/v2/Apps/Conversation/Components/ConversationSnippet.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ const TimeSinceFlex = styled(Flex)`
 `
 
 const StyledSans = styled(Sans)`
-  word-break: break-all;
+  word-break: break-word;
```
Comment (xtina-starr) [inline src/v2/Apps/Conversation/Components/ConversationSnippet.tsx:28]: 👍

File: src/v2/Apps/Conversation/Routes/Conversation/index.tsx (hunk 1 above)
Comment (zephraph) [reply src/v2/Apps/Conversation/Routes/Conversation/index.tsx:27]: It is indeed. Might be good to add a comment here to account for that.

**Pull request merged**